dirty_cat/super_vectorizer.py

```diff
@@ -145,12 +181,13 @@ def __init__(self, *,
                  datetime_transformer: Optional[Union[BaseEstimator, str]] = None,
                  auto_cast: bool = True,
                  impute_missing: str = 'auto',
+                 handle_unknown: str = 'error',
                  # Following parameters are inherited from ColumnTransformer
                  remainder: str = 'passthrough',
                  sparse_threshold: float = 0.3,
                  n_jobs: int = None,
                  transformer_weights=None,
-                 verbose: bool = False,
+                 verbose: bool = False
```

Comment:
It's good practice to leave a comma at the end of an enumeration like this: it means that adding something at the end will require only modifying a single line, and thus decrease the chances of conflict.